### PR TITLE
fix: error msges from 225, and small peer improvement

### PR DESCRIPTION
**Pull request by lesleyrs**

### CODE REVIEW

File: src/js/jagex2/config/FloType.ts [hunk 1]
```diff
@@ -94,7 +94,7 @@ export default class FloType extends ConfigType {
         } else if (code === 6) {
             this.debugname = dat.gjstr;
         } else {
-            console.log('Error unrecognised config code: ', code);
+            throw new Error(`Unrecognised config code: ${code}`);
```
Comment (ultraviolet-jordan) [inline src/js/jagex2/config/FloType.ts:97]: Looking at the Java client for example:
`https://github.com/2004Scape/Client/blob/main/runetek3/src/main/java/jagex2/config/FloType.java#L91`

Does not throw an error.

File: src/js/jagex2/config/IdkType.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export default class IdkType extends ConfigType {
         } else if (code >= 60 && code < 70) {
             this.heads[code - 60] = dat.g2;
         } else {
-            throw new Error(`Unrecognized idk config code: ${code}`);
+            throw new Error(`Unrecognised config code: ${code}`);
```
Comment (ultraviolet-jordan) [inline src/js/jagex2/config/IdkType.ts:47]: Maybe I should have explicitly mentioned but should double check all of these are like the Java client.

Comment (lesleyrs) [reply src/js/jagex2/config/IdkType.ts]: Ah... I actually looked at just the text itself, but it seems they're all just being printed on the java client. Fixed, the mesanim one is made up though.

**Comment by ultraviolet-jordan** [comment]
thank you for fixing my booboo as well

**Pull request merged**